test/bootstrap.js

```diff
@@ -9,14 +9,17 @@ describe('Bootstrap feature', function () {
       helpers.run(path.join(__dirname, '../app'))
         .inDir(path.join(__dirname, 'temp'))
         .withOptions({'skip-install': true})
-        .withPrompts({features: [
-          'includeSass',
-          'includeBootstrap'
-        ]})
+        .withPrompts({
+          features: [
+            'includeSass',
+            'includeBootstrap'
+          ],
+          includeJQuery: false
```

Comment:
I think this option is unnecessary and reads weird, because `includeBootstrap` implies jQuery, because it's a dependency of Bootstrap, while `includeJQuery` is set to false.